src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -669,6 +669,9 @@
   - name: Create a resource policy assignment with a system assigned identity. The identity will have 'Contributor' role access to the subscription.
     text: >
         az policy assignment create --name myPolicy --policy {PolicyName} --assign-identity --identity-scope /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx --role Contributor
+  - name: Create a resource policy assignment with a enforcement mode. The assignment will be enforced with specified enforcement mode (Default or DoNotEnforce).
```

Comment:
Hi, I wonder what enforcement mode mean in `policy assignment` scenario. Can you explain it for me?

Reply:
Basically, it indicates whether policy effect would be enforced at resource creation/update time or not.

The documentation is still in progress, but here is a quick link of review document that explains it in more detail.

[https://review.docs.microsoft.com/en-us/azure/governance/policy/concepts/assignment-structure?branch=pr-en-us-87226#enforcement-mode](https://review.docs.microsoft.com/en-us/azure/governance/policy/concepts/assignment-structure?branch=pr-en-us-87226#enforcement-mode)




---
In reply to: [324499955](https://github.com/Azure/azure-cli/pull/10466#discussion_r324499955) [](ancestors = 324499955)